src/components/Tooltip.scss

```diff
@@ -4,7 +4,7 @@
 	box-shadow: 0 0 20px 0 rgba(26, 26, 26, .3);
 	padding: 12px;
 	position: absolute;
-	z-index: 2;
+	z-index: 1000;
```

Comment:
is it necessary to increase it that much?

Reply:
yes